setup.py

```diff
@@ -14,7 +14,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/algorand/pyteal",
-    packages=["pyteal"],
+    packages=setuptools.find_packages(include=("pyteal*",)),
```

Comment:
@CiottiGiorgio Optional - I think the PR is correct though I believe the following _suggestion_ is more specific.

As is, additional top-level packages (e.g. _pyteal-foo_) will be included.  Based on usage patterns, I _think_ additional top-level packages should opt-in to packaging.

```suggestion
    packages=setuptools.find_packages(include=("pyteal", "pyteal.*",)),
```

Reply:
Absolutely agree. Thanks!